spec/policies/task_policy_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe TaskPolicy do
+  subject { described_class.new(user, task) }
+
+  let(:user) { nil }
```

Comment:
Here (and in some other policy specs above): We might move this line to the `context 'without a user'`.

Reply:
I moved the line and also removed it from the policies that dont have a 'without a user' context anyway.